release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -845,8 +845,8 @@ You can now use the Special Resource Operator (SRO) to help manage the deploymen
 For more information, see xref:../hardware_enablement/psap-special-resource-operator.adoc#about-special-resource-operator_special-resource-operator[About the Special Resource Operator].
 
 [id="ocp-4-9-scale-memory-manager-in-beta-version"]
-==== Memory Manager feature (Technology Preview)
-The Memory Manager feature is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
+==== Memory Manager is generally available
+The Memory Manager, a subcomponent of the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
```

Comment:
the memory manager is a component of the kubelet (https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/) configured using the Performance Addon Operator (up to OCP 4.10) or with the Node Tuning Operator (OCP 4.11 and onwards)

Reply:
@fromanirh @swatisehgal Updated. WDYT?
> The Memory Manager, a kubelet subcomponent configured by the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies: